pygmt/figure.py

```diff
@@ -122,14 +119,8 @@ def _activate_figure(self):
         Start and/or activate the current figure.
 
         All plotting commands run afterward will append to this figure.
-
```

Comment:
Same as above, `gmt figure` also doesn't trigger the generation of a figure file.